content/docs/started/kubeflow-overview.md

```diff
@@ -0,0 +1,151 @@
+shows the workflow stages in sequence, with just one arrow at the end pointing
+back into the flow as an indication of the iterative nature of the process:
+
+<img src="/docs/images/kubeflow-overview-workflow-diagram-1.svg" 
+  alt="A typical machine learning workflow"
+  class="mt-3 mb-3 border border-info rounded">
+
+Looking at the stages in more detail:
+
+* In the experimental phase, you develop your model based on initial
+  assumptions, and test and update the model iteratively to produce the
+  results you're looking for:
+
+  * Identify the problem you want the ML system to solve.
+  * Collect and analyse the data you need to train your ML model.
+  * Choose an ML framework and algorithm, and code the initial version of your 
+    model.
+  * Experiment with the data and with training your model.
+  * Tune the model hyperparameters to ensure the most efficient processing and the
+    most accurate results possible.
+
+* In the production phase, you deploy a system that performs the following 
+  processes:
+
+  * Transform the data into the format that your training system needs. The
+    transformation process must be the same in the experimental and production
+    phases, to ensure that the ML model bases its predictions on the same data 
+    format as that used during training.
+  * Train the ML model.
+  * Serve the model for online prediction or for running in batch mode.
+  * Monitor the model's performance, and feed the results into your processes
+    for tuning or retraining the model.
+
+## Kubeflow components in the ML workflow
+
+The next diagram adds Kubeflow to the workflow, showing which Kubeflow
+components are useful at each stage:
+
+<img src="/docs/images/kubeflow-overview-workflow-diagram-2.svg" 
+  alt="Where Kubeflow fits into a typical machine learning workflow"
+  class="mt-3 mb-3 border border-info rounded">
+
+To learn more, read the following guides to the Kubeflow components:
+
+* Kubeflow includes services for spawning and managing 
+  [Jupyter notebooks](/docs/notebooks/). Use notebooks for interactive data 
+  science and experimenting with ML workflows.
+
+* [Kubeflow Pipelines](/docs/pipelines/pipelines-overview/) is a platform for 
+  building, deploying, and managing multi-step ML workflows based on Docker 
+  containers.
+
+* Kubeflow offers a number of [components](/docs/components/) that you can use
+  to build your ML training, hyperparameter tuning, and serving workloads across
+  multiple platforms.
+
+## Example of a specific ML workflow
+
+The following diagram shows a simple example of a specific ML workflow that you
+can use to train and serve a model trained on the MNIST dataset:
+
+<img src="/docs/images/kubeflow-gcp-e2e-tutorial.svg" 
+  alt="ML workflow for training and serving an MNIST model"
+  class="mt-3 mb-3 border border-info rounded">
+
+For details of the workflow and to run the system yourself, see the 
+[end-to-end tutorial for Kubeflow on GCP](/docs/gke/gcp-e2e/).
```

Comment:
Do we have this example hosted on codelab? An example on codelab would encourage more users to try out Kubeflow immediately - compared to having to set up a cluster by themselves.

Reply:
We do have a codelab for this example, but the codelab runs through the same steps. It'd be great to have a preconfigured environment/sandbox. I think @amygdala is thinking along these lines too.

Reply:
I thought we already had a preconfigured environment, it seems thats not the case yet.
We could eventually have this changed when it is ready.

Reply:
+1